### PR TITLE
Added Russian localization.

### PR DESCRIPTION
**Pull request by YuRaNnNzZZ**

### CODE REVIEW

**Comment by lordjoda** [comment]
Thank you!

File: src/main/resources/assets/opencomputers/lang/ru_RU.lang [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Russian localization by YuRaNnNzZZ
+# Use UTF-8 for this file.
+
+# Blocks
+oc:block.Adapter.name=Адаптер
+oc:block.Cable.name=Кабель
+oc:block.Capacitor.name=Аккумулятор
```
Comment (mishkapp) [inline src/main/resources/assets/opencomputers/lang/ru_RU.lang:7]: "Кондесатор" instead "Аккумулятор", it will be closer to original translation

**Comment by fnuecke** [comment]
If I read this correctly, "Конденсатор" (Кондесатор is a typo, no?) would be equivalent to the German "Kondensator", whereas "Аккумулятор" is more like "Akkumulator"? Yeah, then Конденсатор would be more appropriate I think. I'll change it.

**Comment by mishkapp** [comment]
Yes, these words are similat to each other in German and Russian in writing, pronunciation and sense.

**Pull request merged**